### PR TITLE
fix(groupedInvestigations): display all grouped investigations in table

### PR DESCRIPTION
**Pull request by ItayBenMoshe**
[1156](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1156/)

### CODE REVIEW

**Review by eladyashar — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx [hunk 1]
```diff
@@ -11,30 +11,46 @@ import GroupedInvestigationsForm from './GroupedInvestigationsForm/GroupedInvest
...
         defaultValues: {
-            [GroupedInvestigationsFields.REASON]: groupedInvestigation ?
-                { displayName: groupedInvestigation.groupReason, id: groupedInvestigation.reasonId } : null,
-            [GroupedInvestigationsFields.OTHER_REASON]: groupedInvestigation ? groupedInvestigation.otherReason : ''
+            [GroupedInvestigationsFields.REASON]: null,
+            [GroupedInvestigationsFields.OTHER_REASON]: ''
         }
     });
+    
+    const groupedInvestigation = useMemo(() => {
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:27]: Add Boolean type

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigationsTable/GroupedInvestigationsTable.tsx [hunk 2]
```diff
@@ -15,7 +15,7 @@ const tableHeaders = {
     [TableHeadersNames.statusReason]: 'סטטוס ביצוע'
 };
 
-const GroupedInvestigationsTable = ({ invetigationsToGroup }: Props) => {
+const GroupedInvestigationsTable = ({ investigationsToDisplay }: Props) => {
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigationsTable/GroupedInvestigationsTable.tsx:18]: I think just `investigations` is fine

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts [hunk 3]
```diff
@@ -52,8 +52,7 @@ const useGroupedInvestigations = ({ invetigationsToGroup, onClose }: useGroupedI
             uniqueGroupIds: [],
             epidemiologyNumbers: []
         })
-        if (trimmedGroup.uniqueGroupIds.length === 1
-            && trimmedGroup.epidemiologyNumbers.length > 0) {
+        if (trimmedGroup.uniqueGroupIds.length === 1 && trimmedGroup.epidemiologyNumbers.length > 0) {
```
Comment (NaamanTsur) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts:55]: I know you didn't write this but it should still have a comment or function that explains this.

**Review by RoeiRom — CHANGES_REQUESTED** [review]

**Review by NaamanTsur — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx (hunk 1 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:27]: why boolean?

Comment (RoeiRom) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx:27]: Oops didn’t see that it is being used as an object 🙊

**Review by RoeiRom — DISMISSED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigationsTable/GroupedInvestigationsTable.tsx (hunk 2 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigationsTable/GroupedInvestigationsTable.tsx:18]: fixed

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts (hunk 3 above)
Comment (ItayBenMoshe) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts:56]: fixed

**Review by RoeiRom — APPROVED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts (hunk 3 above)
Comment (NaamanTsur) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts:56]:
```suggestion
        // Check whether there is one grouped investigation and at least one investigation without a group 
        if (trimmedGroup.uniqueGroupIds.length === 1 && trimmedGroup.epidemiologyNumbers.length > 0) {
```

Comment (NaamanTsur) [reply client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/useGroupedInvestigations.ts:56]: better wording

**Review by RoeiRom — APPROVED** [review]

**Review by NaamanTsur — APPROVED** [review]

**Review by RoeiRom — APPROVED** [review]

**Pull request merged**